backend/test/model_test/request_institution_children_test.py

```diff
@@ -0,0 +1,257 @@
+                actual = str(request_institution['key']).encode('utf-8')
+                expected = str(dict_institution['key']).encode('utf-8')
+            else:
+                expected = str(expected_maked_request[dict_key]).encode('utf-8')
+                actual = str(maked_request[dict_key]).encode('utf-8')
+
+            self.assertEqual(actual, expected, "%s is not equal to %s" % (actual, expected))
+
+
+    @patch.object(Invite, 'send_notification')
+    @patch.object(Invite, 'create_notification_message', return_value='mocked_message')
+    def test_send_notification(self, create_notification_message, send_notification):
+        """Test send notification."""
+        request = generate_request(
+            self.admin, self.institution, 
+            self.other_institution
+        )
+
+        request.send_notification(self.institution.key)
+        
+        create_notification_message.assert_called_with(
+            user_key=self.admin.key, 
+            current_institution_key=self.institution.key,
+            sender_institution_key=self.institution.key,
+            receiver_institution_key=self.other_institution.key
+        )
+
+        send_notification.assert_called_with(
+            current_institution=self.institution.key, 
+            receiver_key=self.other_institution.admin, 
+            notification_type='REQUEST_INSTITUTION_CHILDREN',
+            message='mocked_message'
+        )
+
+
+    @patch.object(Invite, 'send_notification')
+    @patch.object(Invite, 'create_notification_message', return_value='mocked_message')
+    def test_send_response_notification(self, create_notification_message, send_notification):
+        """Test send response notification."""
+        request = generate_request(
+            self.admin, self.institution, 
+            self.other_institution
+        )
+
+        for action in ['ACCEPT', 'REJECT']:
+            request.send_response_notification(
+                current_institution=self.other_institution.key, 
+                invitee_key=self.other_institution.admin,
+                action=action
+            )
+            
+            create_notification_message.assert_called_with(
+                user_key=self.other_institution.admin, 
+                current_institution_key=self.other_institution.key,
+                sender_institution_key=self.other_institution.key,
+                receiver_institution_key=self.institution.key
+            )
+
+            notification_type = 'ACCEPT_INSTITUTION_LINK' if action == 'ACCEPT' else 'REJECT_INSTITUTION_LINK'
+            send_notification.assert_called_with(
+                current_institution=self.other_institution.key, 
+                sender_key=self.other_institution.admin, 
+                receiver_key=self.admin.key, 
+                notification_type=notification_type,
+                message='mocked_message'
+            )
+
+
+    @patch.object(EmailSender, 'send_email')
```

Comment:
You gotta mock the method from the entity imported in the model file. So, the correct would be: @patch('models.request_institution_children.RequestLinkEmailSender.send_email')